css/forgetpassword.css

```diff
@@ -0,0 +1,24 @@
+
+body{
+  background:rgba(196,196,196,0.17);
+}
+
+.display{
+  height:200px;
+  background:#1F59BB;
+}
+
+.form-container{
+  position:absolute;
+  padding:30px;
+  background:#ffffff;
+  color:#1F59BB;
+  box-shadow:0px 1px 3px rgba(53,151,245,0.2);
+}
+.form-control{
+  background:rgba(53,151,245,0.05);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5538575)